factory/processFactory.go

```diff
@@ -70,3 +70,12 @@ func CreateEventsInterceptor() (process.EventsInterceptor, error) {
 
 	return process.NewEventsInterceptor(argsEventsInterceptor)
 }
+
+func getPubKeyConverter(cfg *config.GeneralConfig) (core.PubkeyConverter, error) {
```

Comment:
Why don't you use this func in `CreateEventsInterceptor` when creating `pubKeyConverter` ?

Reply:
ahh, forgot to replace, replaced now